airgun/entities/host.py

```diff
@@ -175,6 +171,17 @@ def play_ansible_roles(self, entities_list, timeout=60, wait_for_results=True):
             status_view.wait_for_result(timeout=timeout)
         return status_view.read()
 
+    def delete_hosts(self, entities_list, timeout=60, wait_for_results=True):
+        """Delete all hosts from entities list"""
+        view = self._select_action('Delete Hosts', entities_list)
+        view.submit.click()
+        view.flash.assert_no_error()
+        view.flash.dismiss()
+        status_view = HostsDeleteTaskDetailsView(self.browser)
```

Comment:
you could have directly imported TaskDetailsView here? any reason to wrap with HostsDeleteTaskDetailsView.

Reply:
As you are going from host view to task details view, I think it's good connect these views together by `HostsDeleteTaskDetailsView`.  But I do not know the reason and I did it because two other exemples were defined same way. If we want to change it I can do that :) 

Maybe @ldjebran will know as he implemented these classes: `class HostsJobInvocationCreateView(JobInvocationCreateView):`
why it's important to use new HostViewAndAnotherClassView. 

link to host view entity [here](https://github.com/SatelliteQE/airgun/blob/master/airgun/views/host.py#L619)

Reply:
Okay, I missed looking at those examples. Ok fine